router/terraform/vpc.tf

```diff
@@ -0,0 +1,6 @@
+module "vpc_router" {
+  source     = "github.com/wellcometrust/platform//terraform/network"
```

Comment:
Again this is required as it interacts with the other infra in a sensible way - we have (some) VPCs in a shared stack which might be sensible for you.